person_capture/gui_app.py

```
@@ -8163,6 +8163,24 @@ def _from_center(cx: float, cy: float):
+                try:
+                    candidate_set.add(
+                        self._prefer_lower_face_crop_y(
+                            base_cand,
+                            face,
+                            protect,
+                            bounds,
+                            prof,
+                        )
+                    )
+                except Exception:
+                    pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)